cms/standard_pages/tests/test_models.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from django.utils import timezone
+from django.utils.formats import date_format
+from wagtail.test.utils import WagtailTestUtils
+
+from cms.core.tests.factories import ContactDetailsFactory
+from cms.standard_pages.tests.factories import MethodologyPageFactory
+
+
+class MethodologyPageTestCase(WagtailTestUtils, TestCase):
+    """Test MethodologyPage model properties and methods."""
+
+    def setUp(self):
+        self.basic_page = MethodologyPageFactory(
+            parent__title="Topic Page",
+            title="Methodology Page",
+            published_date=datetime(2024, 8, 15),
+            show_cite_this_page=False,
+            contact_details=None,
+        )
+        self.basic_page_url = self.basic_page.url
+
+        self.page = MethodologyPageFactory(
```

Comment:
You should not need both. Best to start with what you have "basic page" and layer in the tests.